src/main/java/com/airlines/dao/CrewMemberDaoImpl.java

```diff
@@ -0,0 +1,106 @@
+        }
+    }
+
+    private void fillStatementWithCrewMemberData(PreparedStatement insertStatement, CrewMember crewMember)
+            throws SQLException {
+
+        insertStatement.setString(1, crewMember.getFirstName());
+        insertStatement.setString(2, crewMember.getLastName());
+        insertStatement.setString(3, crewMember.getPosition().toString());
+        insertStatement.setDate(4, Date.valueOf(crewMember.getBirthday()));
+        insertStatement.setString(5, crewMember.getCitizenship().toString());
+        insertStatement.setInt(6, crewMember.getCrewId());
+    }
+
+    @Override
+    public void update(CrewMember crewMember) {
+        Objects.requireNonNull(crewMember);
+
+        if (crewMember.getId() == 0) {
+            throw new DaoOperationException("Cannot find a product without ID");
+        }
+
+        try (Connection connection = dbConnector.getDBConnection()) {
+
+            PreparedStatement updateStatement = connection.prepareStatement(UPDATE_CREW_MEMBER_SQL);
+            fillStatementWithCrewMemberData(updateStatement, crewMember);
+            updateStatement.setInt(7, crewMember.getId());
+            int rowsAffected = updateStatement.executeUpdate();
+
+            if (rowsAffected == 0) {
+                throw new DaoOperationException(String.format("Crew member with id = %d does not exist", crewMember.getId()));
+            }
+
+        } catch (SQLException e) {
+            throw new DaoOperationException(String.format("Crew member with id = %d does not exist", crewMember.getId()));
+
+        }
+    }
+
+    @Override
+    public CrewMember findById(int id) {
+        try (Connection connection = dbConnector.getDBConnection()) {
+
+            PreparedStatement selectByIdStatement = connection.prepareStatement(SELECT_CREW_MEMBER_BY_ID_SQL);
+            selectByIdStatement.setInt(1, id);
+            ResultSet resultSet = selectByIdStatement.executeQuery();
+
+            if (resultSet.next()) {
+                CrewMember crewMember = CrewMember.builder()
+                        .withId(resultSet.getInt("id"))
+                        .withCrewId(resultSet.getInt("crew_id"))
+                        .withFirstName(resultSet.getString("first_name"))
+                        .withLastName(resultSet.getString("last_name"))
+                        .withPosition(Position.valueOf(resultSet.getString("position")))
+                        .withBirthday(resultSet.getDate("birthday").toLocalDate())
+                        .withCitizenship(Citizenship.valueOf(resultSet.getString("citizenship")))
+                        .build();
+                return crewMember;
+            } else {
+                throw new DaoOperationException(String.format("Crew member with id = %d does not exist", id));
+            }
+        } catch (SQLException e) {
+            throw new DaoOperationException(String.format("Error finding crew member by id = %d", id));
```

Comment:
You can also rethrow main exception to not loose stracktrace:
 catch (SQLException e) {
            throw new DaoOperationException(String.format("Error finding crew member by _id = %d", id), e);_